pandoc-slave/main.cpp

```diff
@@ -0,0 +1,30 @@
+#include <QCoreApplication>
+
+#include <QDir>
+#include <QCoreApplication>
+#include <QDebug>
+
+#include "pandoc-workflow/pandocrunner.h"
+
+int main(int argc, char *argv[])
+{
+    QCoreApplication a(argc, argv);
+
+    QString projFolder = QCoreApplication::applicationDirPath();
+    QString pandocExePath = QDir(projFolder).filePath("../../pandoc/pandoc.exe");
+    QString inputFile = QDir(projFolder).filePath("../../test-input-files/input.md");
+
+    PandocRunner runner(pandocExePath);
```

Comment:
PandocRunner should have separate setter method that will take pandocExePath. In the future, we could add changing pandoc installation path during runtime.